<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Carlos Antônio Campos Jorge</author>
 <class>Regressao</class>
 <widget class="QMainWindow" name="Regressao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Regressao</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="boxEquacoes">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>181</width>
      <height>421</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Equações</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>177</width>
       <height>398</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label2C">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>9</pointsize>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Dois Coeficientes:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Layout2Coef">
        <item>
         <widget class="QCheckBox" name="checkLinear">
          <property name="text">
           <string>Linear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkPotencial">
          <property name="text">
           <string>Potencial</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkExponencial">
          <property name="text">
           <string>Exponencial</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkLogaritmica">
          <property name="text">
           <string>Logarítmica</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkHiperLinear">
          <property name="text">
           <string>Hiperbólico Linear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkHiperPara">
          <property name="text">
           <string>Hiperbólico Parabólico</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkHill">
          <property name="text">
           <string>Hill</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkAbrams">
          <property name="text">
           <string>Abrams</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkWeibull">
          <property name="text">
           <string>Weibull</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label3C">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>9</pointsize>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Três Coeficientes:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Layout3Coef">
        <item>
         <widget class="QCheckBox" name="checkHillSuav">
          <property name="text">
           <string>Hill Suavizado</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkAbramsSuav">
          <property name="text">
           <string>Abrams Suavizado</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkTalbotRichart">
          <property name="text">
           <string>Talbot-Richart</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="botaoExecuta">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>470</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Executa</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>20</width>
      <height>551</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>20</width>
      <height>551</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabelaEntrada">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>211</width>
      <height>461</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>X</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Y</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="labelY">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelX">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>x:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="campoNomeY">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="campoNomeX">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabelaResultados">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>521</width>
      <height>191</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tabLinear">
     <attribute name="title">
      <string>Linear</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>235</width>
        <height>60</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Valor de &quot;a&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="valorLinearA">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Valor de &quot;b&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="valorLinearB">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabPotencial">
     <attribute name="title">
      <string>Potencial</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>235</width>
        <height>60</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Valor de &quot;a&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="valorPotA">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Valor de &quot;b&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="valorPotB">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabExponencial">
     <attribute name="title">
      <string>Exponencial</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>235</width>
        <height>60</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Valor de &quot;a&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="valorExpA">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Valor de &quot;b&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="valorExpB">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabAbrams">
     <attribute name="title">
      <string>Abrams</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>235</width>
        <height>60</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Valor de &quot;a&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="valorAbramsA">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Valor de &quot;b&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="valorAbramB">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabLog">
     <attribute name="title">
      <string>Logarítmica</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>235</width>
        <height>60</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Valor de &quot;a&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="valorLogA">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Valor de &quot;b&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="valorLogB">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabHiperLinear">
     <attribute name="title">
      <string>Hiperbólico Linear</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>235</width>
        <height>60</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Valor de &quot;a&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="valorHLA">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Valor de &quot;b&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="valorHLB">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabHiperPara">
     <attribute name="title">
      <string>Hiperbólico Parabólico</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>235</width>
        <height>60</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Valor de &quot;a&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="valorHPA">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Valor de &quot;b&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="valorHPB">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabHill">
     <attribute name="title">
      <string>Hill</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>235</width>
        <height>60</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Valor de &quot;a&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="valorHillA">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Valor de &quot;b&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="valorHillB">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabWeibull">
     <attribute name="title">
      <string>Weibull</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>235</width>
        <height>60</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Valor de &quot;a&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="valorWeibullA">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Valor de &quot;b&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="valorWeibullB">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabHillSuav">
     <attribute name="title">
      <string>Hill Suavizado</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>235</width>
        <height>92</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>Valor de &quot;a&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="valorHSA">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>Valor de &quot;b&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="valorHSB">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>Valor de &quot;c&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="valorHSC">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabAbramsSuav">
     <attribute name="title">
      <string>Abrams Suavizado</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>235</width>
        <height>92</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>Valor de &quot;a&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="valorASA">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>Valor de &quot;b&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="valorASB">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>Valor de &quot;c&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="valorASC">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabTalRich">
     <attribute name="title">
      <string>Talbot-Richart</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>235</width>
        <height>92</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="0" column="0">
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>Valor de &quot;a&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="valorTRA">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>Valor de &quot;b&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="valorTRB">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_32">
         <property name="text">
          <string>Valor de &quot;c&quot;:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="valorTRC">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>190</y>
      <width>521</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>210</y>
      <width>501</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionCarregar_arquivo"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCarregar_arquivo">
   <property name="text">
    <string>Carregar arquivo..</string>
   </property>
  </action>
  <action name="actionPlanilha_Excel">
   <property name="text">
    <string>Planilha Excel</string>
   </property>
  </action>
  <action name="actionDocumento_Texto">
   <property name="text">
    <string>Documento Texto</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionLimpar_Tela">
   <property name="text">
    <string>Limpar Tela</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
